vita3k/shader/src/translator/texture.cpp

```diff
@@ -181,11 +181,10 @@ bool USSETranslatorVisitor::smp(
     spv::Id result = do_fetch_texture(sampler, { coord, static_cast<int>(DataType::F32) }, DataType::F32);
 
     switch (sb_mode) {
-    case 1: {
+    case 0:
```

Comment:
```suggestion
    case 0: // TODO: is this correct?
```

Reply:
Not sur i wating @sunho info, but is user like that before is probably same mode 1

Reply:
ok let's see what sunho says

Reply:
Is approuve